test/string_extension_test.dart

```diff
@@ -45,4 +45,49 @@ void main() {
 
     expect(result, isEmpty);
   });
+
+  test('unMarkdownLinks should replace links when there is only one link', () {
+    const messageWithOneLinkUnMarkdowned = "https://superman.com/this-is-a-*superman*-link";
+    const messageWithOneLinkMarkdowned = "https://superman.com/this-is-a-<em>superman</em>-link";
+
+    final result = messageWithOneLinkMarkdowned.unMarkdownLinks(messageWithOneLinkUnMarkdowned);
```

Comment:
what is the input? what is the parameter? hard for me?

Reply:
input is the message with markdown tags such as <em> etc, and parameter is the raw message with no markdown tag

Reply:
how other developer can use your function easily. still hard for me to use. why we know the input when we have a messy message? or smt like that?

Reply:
@imGok maybe you miss this comment

Reply:
It's used when the developer knows that the content of the message may contain links

Reply:
this function is just to prevent links to be unusable because of markdown

Reply:
ok tmr please more detail for me on it. Thanks